builtin/logical/transit/path_encrypt.go

```diff
@@ -159,7 +159,7 @@ func decodeBatchRequestItems(src interface{}, dst *[]BatchRequestItem) error {
 			} else if casted, ok := v.(string); ok {
 				(*dst)[i].Context = casted
 			} else {
-				errs.Errors = append(errs.Errors, fmt.Sprintf("'[%d].context' expected type 'string', got unconvertible type '%T'", i, item["context"]))
+				errs.Errors = append(errs.Errors, fmt.Sprintf("'[%d].context' expected type 'string', got unconvertible type '%T', value: '%v'", i, item["context"], item["context"]))
```

Comment:
How come these changed in this PR? I'm not sure we want to be returning these as errors since they may get logged. Especially the plaintext one below.

Reply:
These changed due to the upgrade in the map structure here: https://github.com/mitchellh/mapstructure/blob/07998cfd7ab2f7f7e5ddce402c4edf38317451eb/mapstructure_examples_test.go#L65